load/loader.go

```diff
@@ -13,13 +13,13 @@ import (
 )
 
 const (
-	// DefaultBatchSize is the default size of batches to be inserted
+	// Default size of batches to be inserted
 	defaultBatchSize = 10000
 	defaultReadSize  = 4 << 20 // 4 MB
 
-	// WorkerPerQueue is the value to have each worker have its own queue of batches
+	// Each worker has its own queue of batches
```

Comment:
Golang doc comments should begin with the name of the item I believe, so these comments are not in line with that.

Reply:
Yes, agreed
Summary: updated